### PR TITLE
Added enemy drops and mana potions

### PR DESCRIPTION
**Pull request by SebastienGorman**
Added mana potions
Fixed bug with enemy drop pouches not being looted properly with stacked items
Enemies now drop a pouch containing 1 hp and 3 mana potions

### CODE REVIEW

File: Assets/Scripts/Inventory/LootableScript.cs [hunk 1]
```diff
@@ -12,15 +12,21 @@ public class LootableScript : BagScript
...
+        AddItem(manaPotion);
+        AddItem(manaPotion);
+        AddItem(manaPotion);
     }
     
     //Debug purposes only
     private void Update()
     {
-        if (Input.GetKeyDown(KeyCode.O))
+        /*if (Input.GetKeyDown(KeyCode.O))
```
Comment (AndreiBaetica) [inline Assets/Scripts/Inventory/LootableScript.cs:26]: let's delete this entirely

**Review by AndreiBaetica — CHANGES_REQUESTED** [review]
bug - all corpses share the same inventory

**Review by RossBaumgartner — APPROVED** [review]
tested and working

**Review by AndreiBaetica — APPROVED** [review]
LGTM

**Pull request merged**